TestAllConvertedEntriesAreSentAndReceivedWithinAnExpectedTimeDuration is unstable

The test failed on CI: https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-collector-contrib/12175/workflows/d114550a-0824-40cd-ac3f-eb4a77852044/jobs/104163

```
--- FAIL: TestAllConvertedEntriesAreSentAndReceivedWithinAnExpectedTimeDuration (0.00s)
    --- FAIL: TestAllConvertedEntriesAreSentAndReceivedWithinAnExpectedTimeDuration/0 (0.12s)
        converter_test.go:279: 
            	Error Trace:	converter_test.go:279
            	Error:      	Max difference between 2021-04-09 20:46:31.0898943 +0000 GMT m=+0.128293201 and 2021-04-09 20:46:31.1047738 +0000 GMT m=+0.143172701 allowed is 10ms, but difference was -14.8795ms
            	Test:       	TestAllConvertedEntriesAreSentAndReceivedWithinAnExpectedTimeDuration/0
    --- FAIL: TestAllConvertedEntriesAreSentAndReceivedWithinAnExpectedTimeDuration/1 (0.12s)
        converter_test.go:279: 
            	Error Trace:	converter_test.go:279
            	Error:      	Max difference between 2021-04-09 20:46:31.090866 +0000 GMT m=+0.129264901 and 2021-04-09 20:46:31.1060946 +0000 GMT m=+0.144493501 allowed is 10ms, but difference was -15.2286ms
            	Test:       	TestAllConvertedEntriesAreSentAndReceivedWithinAnExpectedTimeDuration/1
FAIL
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/internal/stanza	0.180s
FAIL
```

--- a/internal/stanza/converter_test.go
+++ b/internal/stanza/converter_test.go
@@ -210,7 +210,6 @@ func TestAllConvertedEntriesAreSentAndReceivedWithinAnExpectedTimeDuration(t *te
 		entries            int
 		maxFlushCount      uint
 		flushInterval      time.Duration
-		flushTimeTolerance time.Duration
 		expectedNumFlushes int
 	}{
 		{
@@ -218,21 +217,18 @@ func TestAllConvertedEntriesAreSentAndReceivedWithinAnExpectedTimeDuration(t *te
 			maxFlushCount:      20,
 			expectedNumFlushes: 1,
 			flushInterval:      100 * time.Millisecond,
-			flushTimeTolerance: 10 * time.Millisecond,
 		},
 		{
 			entries:            50,
 			maxFlushCount:      51,
 			expectedNumFlushes: 1,
 			flushInterval:      100 * time.Millisecond,
-			flushTimeTolerance: 10 * time.Millisecond,
 		},
 		{
 			entries:            500,
 			maxFlushCount:      501,
 			expectedNumFlushes: 1,
 			flushInterval:      100 * time.Millisecond,
-			flushTimeTolerance: 25 * time.Millisecond,
 		},
 	}
 
@@ -275,8 +271,11 @@ func TestAllConvertedEntriesAreSentAndReceivedWithinAnExpectedTimeDuration(t *te
 						break forLoop
 					}
 
-					tFlushed := time.Now()
-					assert.WithinDuration(t, start.Add(tc.flushInterval), tFlushed, tc.flushTimeTolerance)
+					assert.WithinDuration(t,
+						start.Add(tc.flushInterval),
+						time.Now(),
+						tc.flushInterval,
+					)
 
 					actualFlushCount++
 

